src/app/user-profile/avatar/avatar.component.ts

```diff
@@ -58,20 +56,22 @@ export class AvatarComponent implements OnInit {
         let img = compressedImage
         // Large
         if (bytes > environment.AVATAR_SIZE_LARGE) {
+          console.log('BYTES 1', bytes)
           img = await this.compressByRatio(img, environment.AVATAR_SIZE_LARGE) // limit by service: 110.000
           bytes = this.imageCompress.byteCount(img)
         }
         this.sendImage(img, RefType.Large)
         // Medium
         if (bytes > environment.AVATAR_SIZE_MEDIUM) {
+          console.log('BYTES 2', bytes)
```

Comment:
Christian: do you need such logs anymore?

Reply:
deleted